### DISCUSSION TITLE
All Downloaded videos are saved as getvideo

### DISCUSSION THREAD

**Discussion by Kobra299** [aclap-dev/video-downloadhelper#176] [Help and Q&A] [answered]
### Your question

hello I keep geting my downloaded videos as getvideo instead of the name from the file header and cant change it when I click the download selection any idea on way to fix this please let me known

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.0.23 (stable)
Browser locale: en-GB
Production build
Built on 2024-02-14
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

Companion App installed
Found companion app: VdhCoApp 2.0.10

**Comment by paulrouget** [comment, marked as answer]
> Version 8.2.0.23 (stable)

Hi! This will be fixed in 8.2.1. It will autoupdate very soon!
